tcms/core/contrib/comments/forms.py

```diff
@@ -29,9 +28,7 @@ class SimpleForm(CommentDetailsForm):
         required=False,
     )
 
-    def clean_timestamp(self):
-        return str(time.time()).split('.')[0]
-
+    @staticmethod
```

Comment:
You can safely remove the get_form() method here. The `django_comments` app uses the methods defined in `__init__.py`